code/modules/robotics/robot/module_tool_creator/modules.dm

```diff
@@ -137,12 +137,10 @@
 		///obj/item/reagent_containers/patch/bruise/medbot,
 		/obj/item/reagent_containers/mender/brute/medbot,
 		/obj/item/reagent_containers/mender/burn/medbot,
-		/obj/item/reagent_containers/hypospray, // TODO: make large version
+		/obj/item/robospray, // TODO: make large version
 		/obj/item/reagent_containers/hypospray, // TODO: make large version
 		/obj/item/reagent_containers/syringe,
 		/obj/item/reagent_containers/syringe,
-		/obj/item/reagent_containers/glass/beaker/large/epinephrine,
-		/obj/item/reagent_containers/glass/beaker/large/antitox,
```

Comment:
Considering whether this would work well, thoughts on leaving it out but still keeping two beakers around to allow for player-made chem mixes instead?